packages/services/src/view-service/fees.test.ts

```diff
@@ -88,57 +87,7 @@ describe('extractAltFee', () => {
       ],
     });
 
-    const result = extractAltFee(request);
-    expect(result.equals(swapAssetId)).toBeTruthy();
-  });
-
-  it('extracts the fee from dutchAuctionScheduleActions', () => {
-    const auctionScheduleAssetId = new AssetId({ altBaseDenom: 'auction-schedule' });
-    const request = new TransactionPlannerRequest({
-      dutchAuctionScheduleActions: [
-        new TransactionPlannerRequest_ActionDutchAuctionSchedule({
-          description: { outputId: auctionScheduleAssetId },
-        }),
-      ],
-    });
-
-    const result = extractAltFee(request);
-    expect(result.equals(auctionScheduleAssetId)).toBeTruthy();
-  });
-
-  it('extracts the fee from dutchAuctionEndActions', () => {
-    const auctionEndAuctionId = new AuctionId({ inner: new Uint8Array([3, 2, 5, 2]) });
-    const request = new TransactionPlannerRequest({
-      dutchAuctionEndActions: [
-        new TransactionPlannerRequest_ActionDutchAuctionEnd({
-          auctionId: auctionEndAuctionId,
-        }),
-      ],
-    });
-
-    const result = extractAltFee(request);
-    expect(result.inner).toEqual(auctionEndAuctionId.inner);
-  });
-
-  it('extracts the fee from dutchAuctionWithdrawActions', () => {
-    const auctionWithdrawAuctiontId = new AuctionId({ inner: new Uint8Array([9, 9, 6, 3]) });
-    const request = new TransactionPlannerRequest({
-      dutchAuctionWithdrawActions: [
-        new TransactionPlannerRequest_ActionDutchAuctionWithdraw({
-          auctionId: auctionWithdrawAuctiontId,
-        }),
-      ],
-    });
-
-    const result = extractAltFee(request);
-    expect(result.inner).toEqual(auctionWithdrawAuctiontId.inner);
-  });
-
-  it('throws an error when no asset ID is found', () => {
-    const request = new TransactionPlannerRequest({});
-
-    expect(() => extractAltFee(request)).toThrow(
-      'Could not extract alternative fee assetId from TransactionPlannerRequest',
-    );
```

Comment:
Were these deleted by mistake?

Reply:
will fold them back in when we implement auctions seperately in pr. by passing in the idb handle to `extractAltFees`, we're changing the method behavior so it's expected to necessarily change the tests